grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -623,10 +623,10 @@ class ApiController extends BaseController {
         forward controller: "profile", action: "getLocalImage"
```

Comment:
make sure the value of `fileName` is assigned to parameter `imageId` since getLocalImage action still uses `imageId`